companion/src/firmwares/opentx/opentxeeprom.h

```diff
@@ -98,15 +98,11 @@ class ProtocolsConversionTable: public ConversionTable
         addConversion(PULSES_DSM2, val++);
         addConversion(PULSES_DSMX, val++);
       }
-      if (IS_HORUS_OR_TARANIS(board)) {
+      if (IS_ARM(board)) {
         addConversion(PULSES_CROSSFIRE, val++);
         addConversion(PULSES_MULTIMODULE, val++);
         addConversion(PULSES_PXX_R9M, val++);
-      }
-      else if (IS_ARM(board)) {
-        val++;
```

Comment:
it was for removing CROSSFIRE support from SKY9X but perhaps we could add it soon

Reply:
Yeah, but it only does the mapping. If a protocol is actually availabe is on another place. And on the Sky9x and Taranis the eeprom ids are identical anyway, so I remove that distinction.